src/main/java/com/microsoft/cse/helium/app/controllers/ActorsController.java

```diff
@@ -71,6 +73,9 @@ public Object getAllActors(
           Optional<String> pageSize) {
 
     try {
+      logger.info(String.format("getAllActors (query=%s, pageNumber=%s, pageSize=%s)",
+          query, pageNumber, pageSize));
```

Comment:
Same as above - in general avoid c-style sprintf as we have to keep the type of paramter in sync with the format string. MessageFormat does a better job.